Tell where is libltdl from libtool - was needed to get it built with libtool 2.2.4

---
 configure.ac |   26 +++++++++++++++++++-------
 1 file changed, 19 insertions(+), 7 deletions(-)

--- pulseaudio-0.9.10.orig/configure.ac
+++ pulseaudio-0.9.10/configure.ac
@@ -242,20 +242,32 @@ if test $ret -eq 0 ; then
 else
     AC_MSG_RESULT([no])
 fi
 
 #### libtool stuff ####
-
-AC_LTDL_ENABLE_INSTALL
-AC_LIBLTDL_INSTALLABLE
-AC_LIBTOOL_DLOPEN
-AC_LIBTOOL_WIN32_DLL
+LT_PREREQ([2.2])
+LT_INIT([dlopen win32-dll])
 AC_PROG_LIBTOOL
-AC_SUBST(LTDLINCL)
-AC_SUBST(LIBLTDL)
+#AC_SUBST(LTDLINCL)
+#AC_SUBST(LIBLTDL)
 AC_CONFIG_SUBDIRS(libltdl)
 
+AC_ARG_WITH(libltdl-libs,
+	[  --with-libltdl-libs=DIR  path to libltdl libs], libltdl_libs_prefix=$withval)
+if test x$libltdl_libs_prefix != x; then
+	LIBLTDL="-L${libltdl_libs_prefix}/libltdl.so -lltdl"
+fi
+AC_SUBST(LIBLTDL)
+
+AC_ARG_WITH(libltdl-includes,
+	[  --with-libltdl-includes=DIR  path to libltdl includes], libltdl_prefix=$withval)
+if test x$libltdl_prefix != x; then
+	LTDLINCL="-I${libltdl_prefix} ${libltdl_libs_prefix}/libltdl.so"
+fi
+AC_SUBST(LTDLINCL)
+
+
 old_LIBS=$LIBS
 LIBS="$LIBS $LIBLTDL"
 AC_CHECK_FUNCS([lt_dlmutex_register])
 LIBS=$old_LIBS
 AC_CHECK_TYPES([struct lt_user_dlloader, lt_dladvise], , , [#include <ltdl.h>])
